.nvmrc

```diff
@@ -1 +1 @@
-v18
\ No newline at end of file
+v18.19.0
```

Comment:
Not really related to this PR but it could be good idea upgrade to node version 20 at some point